src/main/resources/db/changelog/changes/v1.1.0/update-drv-item-details-definition.xml

```diff
@@ -0,0 +1,330 @@
+    <update tableName="entity_type_definition">
+      <column name="definition">
+        {
+          "id": "0cb79a4c-f7eb-4941-a104-745224ae0292",
+          "name": "drv_item_details",
+          "private": false,
+          "fromClause": "src_inventory_item LEFT JOIN src_inventory_location effective_location_ref_data ON effective_location_ref_data.id = src_inventory_item.effectivelocationid LEFT JOIN src_inventory_call_number_type call_number_type_ref_data ON call_number_type_ref_data.id::text = ((src_inventory_item.jsonb -> 'effectiveCallNumberComponents'::text) ->> 'typeId'::text) LEFT JOIN src_inventory_call_number_type call_item_number_type_ref_data ON call_item_number_type_ref_data.id::text = (src_inventory_item.jsonb ->> 'itemLevelCallNumberTypeId'::text) LEFT JOIN src_inventory_loclibrary loclib_ref_data ON loclib_ref_data.id = effective_location_ref_data.libraryid LEFT JOIN src_inventory_location permanent_location_ref_data ON permanent_location_ref_data.id = src_inventory_item.permanentlocationid LEFT JOIN src_inventory_material_type material_type_ref_data ON material_type_ref_data.id = src_inventory_item.materialtypeid LEFT JOIN src_inventory_location temporary_location_ref_data ON temporary_location_ref_data.id = src_inventory_item.temporarylocationid JOIN src_inventory_holdings_record hrim ON src_inventory_item.holdingsrecordid = hrim.id JOIN src_inventory_instance instance_details ON hrim.instanceid = instance_details.id",
+          "columns": [
+            {
+              "name": "holdings_id",
+              "dataType": {
+                "dataType": "rangedUUIDType"
+              },
+              "valueGetter": "src_inventory_item.holdingsrecordid",
+              "visibleByDefault": false
+            },
+            {
+              "name": "instance_created_date",
+              "dataType": {
+                "dataType": "dateType"
+              },
+              "valueGetter": "instance_details.jsonb -> 'metadata' ->> 'createdDate'",
+              "visibleByDefault": false
+            },
+            {
+              "name": "instance_id",
+              "dataType": {
+                "dataType": "rangedUUIDType"
+              },
+              "valueGetter": "hrim.instanceid",
+              "visibleByDefault": false
+            },
+            {
+              "name": "instance_primary_contributor",
+              "dataType": {
+                "dataType": "stringType"
+              },
+              "valueGetter": "jsonb_path_query_first(instance_details.jsonb, '$.\"contributors\"[*]?(@.\"primary\" == true).\"name\"'::jsonpath) #>> '{}'::text[]",
+              "visibleByDefault": false
+            },
+            {
+              "name": "instance_title",
+              "dataType": {
+                "dataType": "stringType"
+              },
+              "filterValueGetter": "\"left\"(lower(${tenant_id}_mod_inventory_storage.f_unaccent(instance_details.jsonb ->> 'title'::text)), 600)",
```

Comment:
The big change here (since it's hard to spot in files like this where we just have 1 big new version of the file) is the addition of these `filterValueGetter` properties on some columns. The only other change to the entity type definition is the removal of the `subEntityTypeIds` property (which isn't used anymore)

Reply:
For context, in case someone stumbles across the PR later: the filterValueGetter is the thing we use in the WHERE clause when we convert the FQL query to SQL. This column here illustrates it nicely: there's an index that pretty much exactly matches the `filterValueGetter` here. If we use the `valueGetter` in the WHERE clause, it doesn't have any of the extra processing that is used in the index, so the index isn't used and we end up doing a seq scan in the query plan, which can be extremely slow. But we can't just change the `valueGetter` to resemble the index because then users would see values that are truncated, lowercase, and unaccented. So instead, we have 2 different value getters for rendering (`valueGetter`) vs the behind-the-scenes stuff like filtering and sorting (`filterValueGetter`)